test/node_modules/mapbox-gl-js-test.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+
+var tap = module.exports = require('tap');
+var sinon = require('sinon');
+
+tap.beforeEach(function (done) {
+    this.sandbox = sinon.sandbox.create({
+        injectInto: this,
+        properties: ['spy', 'stub', 'mock']
+    });
+    done();
+});
+
+tap.afterEach(function (done) {
+    this.sandbox.restore();
+    done();
```

Comment:
We may want to call [`window.restore()`](http://github.com/mapbox/mapbox-gl-js/blob/4dc27c3810cb8f09d0de9578ad2e6b0e8eeb499f/js/util/window.js#L8-L9) in here too (unless that causes a perf problem)